src/index.js

```diff
@@ -30,7 +42,12 @@ if (config.hostingEnvironment.applicationInsights) {
   appInsights.setup(config.hostingEnvironment.applicationInsights).start();
 }
 
+
 const init = async () => {
+  localStorage.init({
```

Comment:
Is local storage necessary to add not sure how big the object is expected to be stored in local storage?

Reply:
After inspecting previously added functionality in the other implementations of this in manage and services this was added to the init function and I thought it was required if not happy to remove it?